src/master/filesystem_operations.cc

```diff
@@ -960,6 +960,8 @@ uint8_t fs_mknod(const FsContext &context, uint32_t parent, const HString &name,
 	    fsnodes_quota_exceeded_dir(wd, {{QuotaResource::kInodes, 1}})) {
 		return SAUNAFS_ERROR_QUOTA;
 	}
+	static_cast<FSNodeDirectory *>(wd)->case_insensitive =
```

Comment:
Did you consider to move the responsibility of updating `case_insensitive` attribute to other function, like `fsnodes_fill_attr()`?

Reply:
Considered. But in this case it is useful to keep the implementation on `mkdir `and `mknod ` only in order to specify that these are the places I want to make the set. I consider that `fsnodes_fill_attr` is mainly focused on the attributes than other specific data from nodes and also I consider it won't be necessary to execute this specific code in the other places where `fsnodes_fill_attr` is called. However, thanks to your suggestion I notice an extra place that I was adding this logic and not being completely necessary.